tests/transactional/enable_selinux.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Package: transactional-update
+# Summary: Enable SELinux on transactional server
+#
+# Maintainer: QA-C team <qa-c@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use transactional qw(process_reboot);
+use version_utils qw(is_sle_micro);
+use Utils::Systemd qw(systemctl);
+
+sub run {
+    select_console 'root-console';
+    my $audit_log = "/var/log/audit/audit.log";
+
+    # SELinux is disabled by default in SLE Micro
+    validate_script_output("sestatus", sub { m/SELinux status: .*disabled/ }) if is_sle_micro;
```

Comment:
the is_sle_micro shouldn't be needed?

Reply:
selinux is pre-installed in SLE Micro, but not in microOS... so the command will fail there cause there is no such binary.

Reply:
ah ok in that case I'd use something like `test -d /sys/fs/selinux && test -e /etc/selinux/config`. That's what the `selinuxenabled` command does